packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -15,6 +15,7 @@
                 v-if="shouldShowLabel"
                 :label-for="uniqueId"
                 :is-disabled="isDisabled"
+                :is-visually-required="$attrs.required && isVisuallyRequired"
```

Comment:
I'd be tempted to pull this out as a computed and then you could just pass that computedProp into `form-label` here and the `form-selection-control`.

Kinda subjective though, so if you thought there was more reason to leave it as it is, that's fine too.